app/src/main/java/ch/epfl/sdp/blindly/main_screen/my_matches/MyMatch.kt

```diff
@@ -10,5 +10,6 @@ package ch.epfl.sdp.blindly.main_screen.my_matches
 data class MyMatch(
     var name: String,
     var uid: String,
-    var isExpanded: Boolean
+    var isExpanded: Boolean,
+    var isDeleted: Boolean
```

Comment:
Yep, practical to add a flag in order to distinct deleted users and not completely destroy chat history.